integration/shareddata/composites.go

```diff
@@ -52,39 +52,66 @@ var Composites = &Values[string]{
 var DocumentsDoubles = &Values[string]{
 	name:     "DocumentsDoubles",
 	handlers: []string{"pg", "tigris"},
+	validators: map[string]map[string]any{
+		"tigris": {
+			"$tigrisSchemaString": tigrisSchema(`"type": "object", "properties": {"v": {"type": "number"}}`),
+		},
+	},
 	data: map[string]any{
 		"document-double":          bson.D{{"v", 42.13}},
 		"document-double-whole":    bson.D{{"v", 42.0}},
 		"document-double-zero":     bson.D{{"v", 0.0}},
 		"document-double-max":      bson.D{{"v", math.MaxFloat64}},
 		"document-double-smallest": bson.D{{"v", math.SmallestNonzeroFloat64}},
 		"document-double-big":      bson.D{{"v", doubleBig}},
-		// TODO Dealing with empty doc needs a schema to be defined https://github.com/FerretDB/FerretDB/issues/772
-		// "document-empty":           bson.D{},
+		"document-double-empty":    bson.D{},
+		"document-double-nil":      nil,
```

Comment:
```suggestion
		"document-double-null":      nil,
```

That's the suffix we use in our data sets